AlamofireXMLRPC.xcodeproj/project.pbxproj

```diff
@@ -518,6 +601,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5E513F774BDB1750A705D36C /* Pods-AlamofireXMLRPC.debug.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
```[!] The `AlamofireXMLRPC [Debug]` target overrides the `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` build setting defined in `Pods/Target Support Files/Pods-AlamofireXMLRPC/Pods-AlamofireXMLRPC.debug.xcconfig'. This can lead to problems with the CocoaPods installation```